mqtt/mqtt-edgehub/src/auth/authentication/edgehub.rs

```diff
@@ -74,26 +74,20 @@ impl Authenticator for EdgeHubAuthenticator {
         &self,
         context: AuthenticationContext,
     ) -> Result<Option<AuthId>, Self::Error> {
-        let authenticate = || async {
+        // try to authenticate a client. it retries with 500ms interval until
+        // it gives up after 1min of trying.
+        let when_stop_attempts = Instant::now() + Duration::from_secs(60);
```

Comment:
no more exponential backoff???

Reply:
So I am not really sure what the pro/con situation here is, so maybe you or @dmolokanov can comment... as at least from comments we are looking at the difference between 10 attempts and 120 attempts (60s / 500ms)... Which seems like a lot, but maybe it is justified? IDK...

Reply:
it was an ongoing discussion if we even need a retries here: it started long before we had special readiness edgehub event. So I simplified code here and removed one dependency. that's what @vipeller also was in favor of!

Reply:
hm... true, with all those changes to "broker startup", it is "very likely" that auth endpoint is already available.